pytorch_toolkit/formula_recognition/tests/test_export.py

```diff
@@ -0,0 +1,135 @@
+            target_metric = evaluator.expected_outputs.get("target_metric")
+            self.assertGreaterEqual(metric_onnx, target_metric)
+
+        def test_4_encoder_ir_export(self):
+            if not self.config.get("export_ir"):
+                return
+            self.exporter.export_model_if_not_yet(model=self.config.get(
+                "res_encoder_name"), model_type='encoder', ir=False)
+            encoder_res_name = self.config.get("res_encoder_name").replace('onnx', 'xml')
+            result_model_exists = os.path.exists(encoder_res_name)
+            if result_model_exists:
+                os.remove(encoder_res_name)
+            encoder_res_name = self.config.get("res_encoder_name").replace('onnx', 'bin')
+            result_model_exists = os.path.exists(encoder_res_name)
+            if result_model_exists:
+                os.remove(encoder_res_name)
+
+            self.exporter.export_encoder_ir()
+            encoder_res_name = self.config.get("res_encoder_name").replace('onnx', 'bin')
+            result_model_exists = os.path.exists(encoder_res_name)
+            self.assertEqual(True, result_model_exists)
+            encoder_res_name = self.config.get("res_encoder_name").replace('onnx', 'xml')
+            result_model_exists = os.path.exists(encoder_res_name)
+            self.assertEqual(True, result_model_exists)
+
+        def test_5_decoder_ir_export(self):
+            if not self.config.get("export_ir"):
+                return
+            self.exporter.export_model_if_not_yet(model=self.config.get(
+                "res_decoder_name"), model_type='decoder', ir=False)
+            decoder_res_name = self.config.get("res_decoder_name").replace('onnx', 'xml')
+            result_model_exists = os.path.exists(decoder_res_name)
+            if result_model_exists:
+                os.remove(decoder_res_name)
+            decoder_res_name = self.config.get("res_decoder_name").replace('onnx', 'bin')
+            result_model_exists = os.path.exists(decoder_res_name)
+            if result_model_exists:
+                os.remove(decoder_res_name)
+
+            self.exporter.export_decoder_ir()
+            decoder_res_name = self.config.get("res_decoder_name").replace('onnx', 'bin')
+            result_model_exists = os.path.exists(decoder_res_name)
+            self.assertEqual(True, result_model_exists)
+            decoder_res_name = self.config.get("res_decoder_name").replace('onnx', 'xml')
+            result_model_exists = os.path.exists(decoder_res_name)
+            self.assertEqual(True, result_model_exists)
+
+        def test_6_run_ir_model(self):
+            if not self.config.get("export_ir"):
+                return
+            for model_type in ('encoder', 'decoder'):
+                for ir in (False, True):
+                    self.exporter.export_model_if_not_yet(model=self.config.get(
+                        f"res_{model_type}_name"), model_type=model_type, ir=ir)
```

Comment:
```suggestion
                self.exporter.export_model_if_not_yet(model=self.config.get(
                    f"res_{model_type}_name"), model_type=model_type, ir=True)
```
-- now  `export_model_if_not_yet` with `ir=True` will run `export_model_if_not_yet` with `ir=False` -- if required

Reply:
I've made `export_model_if_not_yet` private function and added corresponding wrappers (`export_to_onnx_model_if_not_yet` and `export_to_ir_model_if_not_yet`)